cmd/helpMessages.go

```diff
@@ -172,6 +172,12 @@ Log in using a VM's user-assigned identity with an Object ID of the service iden
 Log in using a VM's user-assigned identity with a Resource ID of the service identity:
 - azcopy login --identity --identity-resource-id "/subscriptions/<subscriptionId>/resourcegroups/myRG/providers/Microsoft.ManagedIdentity/userAssignedIdentities/myID"
 
+Log in as a service principal using a client secret:
```

Comment:
How about making these span two lines, putting the setting of the env on a line of its own, then the AzCopy command on the next line.